src/store/dicom-web/dicom-web-store.ts

```diff
@@ -43,10 +45,17 @@ async function getAllPatients(host: string): Promise<PatientInfo[]> {
  * Collect DICOM data from DICOMWeb
  */
 export const useDicomWebStore = defineStore('dicom-web', () => {
-  const host = process.env.VUE_APP_DICOM_WEB_URL
-    ? ref(process.env.VUE_APP_DICOM_WEB_URL)
-    : useLocalStorage<string>('dicomWebHost', '');
-  const isConfigured = computed(() => !!host.value);
+  const urlParams = vtkURLExtract.extractURLParameters() as UrlParams;
+  const dicomWebFromURLParam = urlParams.dicomweb as string | undefined;
+  const initialHost = dicomWebFromURLParam ?? process.env.VUE_APP_DICOM_WEB_URL;
+
+  const host = initialHost
+    ? ref(initialHost)
+    : useLocalStorage<string | null>('dicomWebHost', ''); // null if cleared by vuetify text input
```

Comment:
Should `initialHost` also be saved to local storage, or do you think the `dicomweb` parameter should be an overriding exception?